conf/application.conf

```diff
@@ -59,6 +59,7 @@ webKnossos {
   cache {
     user.timeout = 3 minutes
   }
+  annotation.mutex.expiryTime = 2 minutes
```

Comment:
@fm3 As this is a setting that might be changed, I think it would be a good idea to expose it to the frontend. Thus the frontend can adjust to the timeout duration so it requests the mutex in time.

If that setting is not designed to be changed, ignore my post 📰